documentation/plugin/customisation.rst

```diff
@@ -81,6 +81,35 @@ get computed by your custom function! For later lookup, the data will be linked
 .. todo:: We're planning to use a similar approach to allow for custom forecasting algorithms, as well.
 
 
+Deploying your plugin via Docker
+----------------------------------
+
+You can extend the FlexMeasures Docker image with your plugin's logic.
```

Comment:
I would mention that this procedure allows to add multiple FM plugins in a single docker image.